frontend/src/v5/helpers/viewpoint.helpers.ts

```diff
@@ -20,11 +20,8 @@ import { generateViewpoint as generateViewpointV4, getNodesIdsFromSharedIds, toS
 import { formatMessage } from '@/v5/services/intl';
 import { dispatch, getState } from '@/v4/modules/store';
```

Comment:
- Select a ticket that has a default view with some transformations;
- Go to that ticket's groups;
- Clear transformations;
- Create a new group;

This will set the transformations back to what's in the ticket default view, is this a bug?